Dropdown/Dropdown.module.less

```diff
@@ -238,6 +238,8 @@
 .applySkins({
 	.contextualPopup{
 		.container {
+			background-color: @agate-dropdown-list-bg-color;
+			border-radius: @agate-button-border-radius;
```

Comment:
Would you please tell me why you change this from `border-radius: @agate-dropdown-list-border-radius;`?

Reply:
I think when I tried the fixes I forgot it by mistake like that. I'm sorry.